worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -162,26 +160,31 @@ private boolean filterByYear(IDomainResource patient) {
      * @return patientId if present, null otherwise
      */
     private Identifiers extractPatientId(IDomainResource patient) {
+        List<PatientIdentifier> ids = IdentifierUtils.getIdentifiers(patient);
         // Get patient beneficiary id
         // if not found eobs cannot be looked up so do not return a meaningful list
-        String beneId = IdentifierUtils.getBeneId(patient);
-        if (beneId == null || beneId.isEmpty()) {
+        PatientIdentifier beneIdObj = IdentifierUtils.getBeneId(ids);
+        if (beneIdObj == null || beneIdObj.getValue() == null || beneIdObj.getValue().isEmpty()) {
```

Comment:
You can use Apache StringUtils to test if the string is blank `StringUtils.isBlank(beneIdObj.getValue())`